examples/headsail-bsp/src/sysctrl/soc_ctrl.rs

```diff
@@ -115,6 +115,14 @@ pub fn clk3_mask(conf_val: u32) {
     mask_u32(mmap::SS_CLK_CTRL3, conf_val);
 }
 
+pub fn sdram_cfg_axi_ddr_mode_mask(conf_val: u32) {
+    mask_u32(mmap::SDRAM_CFG_AXI_DDR_MODE_ADDR, conf_val);
+}
+
+pub fn sdram_cfg_axi_enable_mask(conf_val: u32) {
+    mask_u32(mmap::SDRAM_CFG_AXI_ENABLE_ADDR, conf_val);
+}
+
```

Comment:
A new file "sdram.rs" is likely the correct place for these. Add `pub mod sdram;` in lib.rs to expose it.

Reply:
Created `sdram.rs` and moved these functions there. Exposed `sdram` in lib.rs